seed/utils/geocode.py

```diff
@@ -128,15 +148,18 @@ def _full_address(building):
     Before passing the address back, special and reserved characters are removed.
     """
 
-    address_components = [
-        building.address_line_1 or "",
-        building.address_line_2 or "",
-        building.city or "",
-        building.state or "",
-        building.postal_code or ""
-    ]
+    address_components = []
+    for col in geocoding_columns:
+        if col['is_extra_data']:
+            address_value = building.extra_data.get(col['column_name'], None)
+        else:
+            address_value = getattr(building, col['column_name'])
+
+        # Only accept non-empty strings or numbers
+        if (isinstance(address_value, (str, Number))) and (address_value != ""):
+            address_components.append(str(address_value))
 
-    if address_components.count("") < 3:
+    if len(address_components) > 2:
         full_address = ", ".join(address_components)
         return re.sub(r'[;/?:@=&"<>#%{}|["^~`\]\\]', '', full_address)
```

Comment:
there a test for this one? I'm sure I'll run into it as i'm going through it...

Reply:
https://github.com/SEED-platform/seed/pull/1986/files#diff-10e49244e46b722e692f9065896cc2faR340

Note: Your comment was coming in on a different commit than the latest commit, but I think you were still referring to the `len(address_components > 0:` version.

<img width="1396" alt="Screen Shot 2019-11-04 at 1 00 24 PM" src="https://user-images.githubusercontent.com/30608004/68154525-ec3ec700-ff04-11e9-85c3-89bdd3c9e82a.png">
(non-latest comment on the left)